rdmo/projects/rules.py

```diff
@@ -76,7 +76,6 @@ def is_site_manager(user, project):
 rules.add_perm('projects.change_value_object', is_project_author | is_project_manager | is_project_owner | is_site_manager)
 rules.add_perm('projects.delete_value_object', is_project_author | is_project_manager | is_project_owner | is_site_manager)
 
-rules.add_perm('questions.view_questionset_object', is_project_member | is_site_manager)
```

Comment:
Do we need that? Currently everyone who is logged in can see any questionset, right?

Reply:
I don't know, this permission was there and I thought to just move it to the `questions/rules.py` app because it applies to the `questions` model. Im not sure when this rule is used..

Reply:
Ah, sorry, I thought you added it. I checked, it is actually needed to check if a user can access `ProjectQuestionSetViewSet` since it checks conditions of the users project. I added it a while ago and it needs to stay in `rdmo/projects/rules.py`. Maybe we need to rename it if it clashes with the new rules.

Reply:
yes it clashes with the name, Ive added now ` | is_element_editor` (imported from `questions`) to this rule. Rename will be more difficult because the `core/permissions` need to be changed for that?

Reply:
```py
  File "../rdmoorganiser/rdmo/rdmo/projects/rules.py", line 7, in is_project_member
    return user in project.member or (project.parent and is_project_member(user, project.parent))
                   ^^^^^^^^^^^^^^
AttributeError: 'QuestionSet' object has no attribute 'member'
```

now I see, this brings another problem that the `QuestionSet` and/or `Project` object are both checked with the rules..

I can fix it with 
```py
if not hasattr(obj, 'member'):
    return False
```
for example

Reply:
Ive added another check:
```py
if 'project' in obj._meta.model_name
```

Reply:
was resolved with 
```py
rules.add_perm('projects.view_questionset_object', is_project_member | is_site_manager)
```